apps/gazes-api/src/app/routes/animes.ts

```diff
@@ -1,93 +1,49 @@
-import {
-	type AnimeListQuerystring,
-	AnimeListQuerystringSchema,
-	type AnimeDetailParams,
-	AnimeDetailParamsSchema,
-	type EpisodeParams,
-	EpisodeParamsSchema,
-} from "@api/contracts/animesContract";
-import type { FastifyInstance } from "fastify";
-import type { AppOptions } from "@api/main";
+import { AppOptions } from "@api/main";
+import { FastifyInstance } from "fastify";
 import { AnimeService } from "../services/animeService";
-
-/**
- * Initializes anime listing and detail routes.
- *
- * @param {FastifyInstance} fastify - The Fastify instance.
- * @param {AppOptions} options - The application options including Redis and Prisma clients.
- */
-export default async function (fastify: FastifyInstance, { redis, prisma }: AppOptions) {
-	const animeService = new AnimeService(prisma, redis);
-
-	/**
-	 * Route serving a paginated list of animes with optional filtering.
-	 *
-	 * @route GET /animes
-	 * @param {AnimeListQuerystring} req.query - Filters for the anime list.
-	 * @returns {void}
-	 */
-	fastify.get<{ Querystring: AnimeListQuerystring }>("/animes", { schema: { querystring: AnimeListQuerystringSchema } }, async (req, rep) => {
-		const { page = 1, title, genres, status, releaseDate } = req.query;
-		const queryOptions = {
-			skip: 25 * (page - 1),
-			take: 25,
-			where: {},
-		};
-
-		if (title) queryOptions.where["others"] = { search: title.split(" ").join(" & ") };
-		if (genres) queryOptions.where["genres"] = { hasEvery: genres.split(",") };
-		if (status) queryOptions.where["status"] = { equals: status.toString() };
-		if (releaseDate)
-			queryOptions.where["start_date_year"] = {
-				equals: releaseDate.toString(),
-			};
-
-		const animeList = await prisma.anime.findMany(queryOptions);
-		rep.status(200).send(animeList);
-	});
-
-	/**
-	 * Route serving the latest episodes of animes.
-	 *
-	 * @route GET /animes/latests
-	 * @returns {void}
-	 */
-	fastify.get("/animes/latests", async (req, rep) => {
-		try {
-			const cachedLatestList = await redis.get("latests");
-
-			if (cachedLatestList) {
-				rep.status(200).send(JSON.parse(cachedLatestList));
-				return;
+import { AnimeDetailParams, AnimeDetailParamsSchema, AnimeListQuerystring, AnimeListQuerystringSchema } from "@api/contracts/animesContract";
+import { StatusCodes } from "http-status-codes";
+import { Anime, Prisma } from "@prisma/client";
+import { CacheManager } from "../utils/cacheManager";
```

Comment:
Try not to use relative import, if you move the file or implementation, you can break it